# Position Controllers ---------------------------------------
joint1_position_controller:
  command_topic: joint1_velocity_controller/command
  joint: joint1
  pid: {p: 15.0, i: 40.0, d: 0.0, high_limit: 6, low_limit: -6}
joint2_position_controller:
  command_topic: joint2_velocity_controller/command
  joint: joint2
  pid: {p: 10.0, i: 30.0, d: 0.0, high_limit: 6, low_limit: -6}
joint3_position_controller:
  command_topic: joint3_velocity_controller/command
  joint: joint3
  pid: {p: 5.0, i: 20.0, d: 0.0, high_limit: 6, low_limit: -6}